tests/benchdnn/dnnl_common.cpp

```diff
@@ -30,6 +30,14 @@
 #include "oneapi/dnnl/dnnl_sycl.hpp"
 #endif
 
+#if DNNL_GPU_VENDOR == DNNL_VENDOR_INTEL
+#define NGEN_NO_OP_NAMES
+#include "ngen/ngen_opencl.hpp"
```

Comment:
As an option this query could be implemented in the library, e.g. see `dnnl_impl_gpu_mayiuse_ngen_kernels`. This way we could avoid benchdnn dependency on nGEN.

@dzarukin @mgouicem Please weigh in.